tests/utils.go

```diff
@@ -478,7 +478,7 @@ func WaitForAllPodsReady(timeout time.Duration, listOptions metav1.ListOptions)
 		virtClient, err := kubecli.GetKubevirtClient()
 		PanicOnError(err)
 
-		podsList, err := virtClient.CoreV1().Pods(k8sv1.NamespaceAll).List(listOptions)
+		podsList, err := virtClient.CoreV1().Pods(KubeVirtInstallNamespace).List(listOptions)
```

Comment:
To have the pods Running/Ready in all namespaces doesn't seem ideal to me, but we should have more checks to see if core functional pods are fine.
But it this is a hotfix, I would lean towards lgtm